spec/features/new_project_spec.rb

```diff
@@ -223,6 +223,43 @@
     expect(File).to exist("#{project_path}/spec/factories.rb")
   end
 
+  it "creates the local heroku setup" do
+    bin_setup_path = "#{project_path}/bin/setup_review_app"
+    bin_setup = IO.read(bin_setup_path)
+
+    expect(bin_setup).to include("heroku run rake db:migrate --exit-code --app #{app_name.dasherize}-staging-pr-$1")
+    expect(bin_setup).to include("heroku ps:scale worker=1 --app #{app_name.dasherize}-staging-pr-$1")
+    expect(bin_setup).to include("heroku restart --app #{app_name.dasherize}-staging-pr-$1")
+    expect(File.stat(bin_setup_path)).to be_executable
+
+    bin_deploy_path = "#{project_path}/bin/deploy"
+    bin_deploy = IO.read(bin_deploy_path)
+
+    expect(bin_deploy).to include("heroku run rake db:migrate --exit-code")
+    expect(File.stat(bin_deploy_path)).to be_executable
+
```

Comment:
> it "creates a circle.yml file with staging deployment commands" do

Now that I think of this, it will fail if there is no Heroku app set up. Circle CI configuration should only happen with `--heroku true` flag.